test.sh

```diff
@@ -79,6 +79,28 @@ function check_gold() {
   fi
 }
 
+function check_error() {
```

Comment:
What's the reason for introducing `check_error` instead of using `check_gold`?

Reply:
I did not see that there were already tests for the error reporting. I reverted the commit and fixed it. I apologize for that.